.gitignore

```diff
@@ -7,6 +7,7 @@ cargo-timing*.html
 .DS_Store
 .idea/
 .vscode/
+.helix/
```

Comment:
Not the biggest deal, especially since there is precedent here already, but I like to think of `.gitignore` only have project-specific ignore patterns and that users can configure their global gitignore config to ignore environment / editor specific files. This avoids the need to add ignore patterns to every project that one interacts with.

Reply:
That is a good philosophy to follow. I didn't think about that when I saw the vscode one...
I'm happy to remove it!

Reply:
🤔 given we have vscode and idea in there already, if we remove helix I'd suggest removing those two as well?

Reply:
opened https://github.com/vectordotdev/vector/pull/17267